### PR TITLE
Add relevant parametrs for RGI

### PR DESCRIPTION
**Pull request by Darcy220606**
This is to address issue #80

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -56,6 +56,11 @@ params {
 
     arg_skip_rgi                        = false
     arg_rgi_savetmpfiles                = false
+    arg_rgi_alignmenttool               = 'BLAST'
+    arg_rgi_includeloose                = false
```
Comment (louperelo) [inline nextflow.config:60]: The RGI docs say about the `loose` algorithm, that "Combined with phenotypic screening, the Loose algorithm allows researchers to hone in on new AMR genes.". Shouldn't we include it by default to get the highest number of possible results?
```suggestion
    arg_rgi_includeloose                = true
```

File: nextflow_schema.json [hunk 2]
```diff
@@ -374,6 +374,40 @@
...
+                    "default": "BLAST",
+                    "description": "Specifies the alignment tool to be used.",
+                    "help_text": "Specifies the alignment tool to be used. By default the tool used for alignment is `DIAMOND`. This flag activates `BLAST` instead.\n\n> Modifies tool parameter(s):\n> - RGI: `--alignment_tool`",
+                    "enum": ["BLAST", "DIAMOND"],
+                    "fa_icon": "fas fa-align-justify"
+                },
+                "arg_rgi_includeloose": {
+                    "type": "boolean",
+                    "description": "Includes all of loose, strict and perfect hits (i.e. >=95% identity) found by rgi.",
+                    "help_text": "When activated it includes all of loose, strict and perfect hits (i.e. also those <=95% identity) found by rgi, and list those hits still as 'strict'.\n\n> Modifies tool parameter(s):\n> - RGI: `--include_loose`",
```
Comment (louperelo) [inline nextflow_schema.json:389]: The behaviour that RGI includes the loose hits with an identity of 95% or better as `strict` is from versions previous to 6.0.0. according to the documentation on git (https://github.com/arpcard/rgi#help-menus-for-subcommands). The module still uses v.5.2.1.  Should we update the module at some point?

```suggestion
                    "help_text": "When activated it includes loose hits (a.k.a. Discovery) in addition to strict and perfect hits. All Loose matches of 95% identity or better are automatically listed as Strict, regardless of alignment length (RGI v. <6.0.0). This behaviour can be overrun by using the --exclude_nudge flag. The Loose algorithm works outside of the detection model cut-offs to provide detection of new, emergent threats and more distant homologs of AMR genes, but will also catalog homologous sequences and spurious partial matches that may not have a role in AMR  \n\n> Modifies tool parameter(s):\n> - RGI: `--include_loose`",
```

File: nextflow.config [hunk 3]
```diff
@@ -56,6 +56,11 @@ params {
...
+    arg_rgi_excludenudge                = false
```
Comment (louperelo) [inline nextflow.config:61]: If so, then also set `--exclude_nudge` to true in order to keep Loose, Strict and Perfect hits separate in the output.
```suggestion
    arg_rgi_excludenudge                = true
```

**Review by louperelo — APPROVED** [review]
The code looks good. RGI extra commands did not seem to be an easy task ... I made some suggestions, maybe we discuss in the meeting tomorrow?

File: nextflow_schema.json (hunk 2 above)
Comment (Darcy220606) [reply nextflow_schema.json:389]: Indeed, it seems like they made a major update of RGI, thats why I made an issue for that. Perhaps after the first release?

**Pull request merged**